<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeDeDialog</class>
 <widget class="QDialog" name="SeDeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询和删除操作</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 127);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设备名称：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bdel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除设备</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sbname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Bsel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BdelWx">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除维修信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BselWx">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询维修信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BdelBf">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除报废信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BselBf">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询报废设备</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
